circuit/src/rns/secp256k1.rs

```diff
@@ -0,0 +1,222 @@
+use super::*;
+
+#[derive(Debug, Clone, PartialEq, Default)]
+/// We implement two structs one for the Secpk256k1 Base Field as the wrong field and one for the Secp256k1 Scalar Field as the wrong field. The native field is the BN256 scalar field.
+/// The reason for implementing both these structs is that ECDSA verification contains operations in both fields.
+/// Struct for the Secp256k1 Base Field as the wrong field. https://github.com/privacy-scaling-explorations/halo2curves/blob/main/src/secp256k1/fp.rs
+///
```

Comment:
Can you format these lines such that they don't pass 100 chars,
not sure why this is not handled by the formatter